components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/RequestObjectValidatorImpl.java

```diff
@@ -231,7 +144,9 @@ private boolean checkExpirationTime(RequestObject requestObject) throws RequestO
                         ", Expiration Time(ms) : " + expirationTimeInMillis +
                         ", TimeStamp Skew : " + timeStampSkewMillis +
                         ", Current Time : " + currentTimeInMillis + ". Token Rejected.";
-                logAndReturnFalse(msg);
+                if (log.isDebugEnabled()) {
+                    log.debug(msg);
```

Comment:
'msg' should be built inside this block only when debug enabled

Reply:
this is fixed